changelogs/fragments/update_supported_versions.yaml

```diff
@@ -0,0 +1,3 @@
+---
+trivial:
```

Comment:
I'm not sure though, but, is this a trivial change?

Reply:
Yeah, the changes to the python library version and kubernetes version should be breaking changes.